frontend/cypress/downloads/users_2022-08-16T18_23_34.553Z.csv

```diff
@@ -0,0 +1,2 @@
+﻿"First Name","Last Name","Veteran","Citizenship","Language","Bilingual Evaluation","Comprehension Level","Written Level","Verbal Level","Estimated Language Ability","Government Employee","Department","Employee Type","Lateral Deployment / Secondment","Current Classification","Location Preferences","Location Exemptions","Accept Temporary","Accepted Operation Requirements","Woman","Indigenous","Visible Minority","Disabled","Role/Salary Expectation","Skills"
```

Comment:
Does this need to be version controlled? I ran `npm run e2e:run` and it deleted this file and created a new one:
![Screen Shot 2022-08-23 at 15 26 00](https://user-images.githubusercontent.com/3046459/186248811-cf6dbb93-bdd9-492c-836f-1d190215f95c.png)

Reply:
Dang, didn't realize I missed this in the `.gitignore`. We are now ignoring it: [don't version control cypress downloads](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3695/commits/6b3de15f23a8688d4c84a5ab366825c909c4f4eb)